app/presenters/user_presenter.rb

```diff
@@ -1,4 +1,6 @@
 class UserPresenter < Burgundy::Item
+  include ActionView::Helpers
```

Comment:
This is a sign that there's some logic in here that belongs in a view. I would posit that any method that depends on `ActionView::Helpers` be done directly in the model.